packages/documentation/src/stories/components/choice-card/radiobutton-card.docs.mdx

```diff
@@ -38,6 +38,33 @@ import * as RadiobuttonCardStories from './radiobutton-card.stories';
   <Controls of={RadiobuttonCardStories.Default} />
 </div>
 
+## Firefox fallback snippet
+
+This snippet adds a global event listener to mirror the `focused` and `checked` states to the parent of the input. This fallback has to be applied as long as Firefox does not support the `:has` selector.
+
+<Source
+  code={`
+['focusin', 'focusout', 'input'].forEach(t => document.addEventListener(t, e => {
+  if (!(e.target instanceof Element) || e.target.nodeName !== 'input') return; 
+  const parent = e.target.parentElement;
+  if (!parent?.classList.contains('choice-card')) return;
+  switch (e.type) {
+    case 'focusin':
+      parent.classList.add('focused');
+      break;
+    case 'focusout':
+      parent.classList.remove('focused');
+      break;
+    case 'input':
+      parent.classList.toggle('checked', (e.target as HTMLInputElement).checked);
+      break;
+  }
+}));
+`}
+  language="typescript"
+  dark="true"
```

Comment:
I think we said that we don't use dark mode as it's not supported by all type of code snippet?